### PR TITLE
Re-org cloud section and add top level getting started. Fixes #67, #69

### LINKED ISSUE #67: Create top-level Quickstart
We want to add a new section make "Quickstart" and add a link to the nav. Right now quickstart material is in more than one section. This would consolidate them and make them easier to discover by being in the navbar.

### PR DESCRIPTION
**Pull request by ChrisGammell**

### CODE REVIEW

File: docs/hardware/3-esp32/2-quickstart/1-overview.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: /hardware/esp32/quickstart
 ---
 
 :::caution
-You must first follow the [Golioth Platform Quickstart](/services/getting-started/) before attempting this guide.
+You must first follow the [Golioth Platform Quickstart](/getting-started/) before attempting this guide.
```
Comment (0Grit) [inline docs/hardware/3-esp32/2-quickstart/1-overview.md:8]: My first trip through this I actually got confused whether I was doing the quickstart or the getting started guide. Then I realized they were one and the same eventually.

File: docs/hardware/4-nrf91/2-quickstart/1-overview.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ slug: /hardware/nrf91/quickstart
 ---
 
 :::caution
-You must first follow the [Golioth Platform Quickstart](/services/getting-started) before attempting this guide.
+You must first follow the [Golioth Platform Quickstart](/getting-started) before attempting this guide.
```
Comment (0Grit) [inline docs/hardware/4-nrf91/2-quickstart/1-overview.md:8]: See my quickstart/getting started comment above

File: docs/hardware/5-virtual-device/2-quickstart/1-overview.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ slug: /hardware/virtual-device/quickstart
 ---
 
 :::caution
-You must first follow the [Golioth Platform Quickstart](/services/getting-started) before attempting this guide.
+You must first follow the [Golioth Platform Quickstart](/getting-started) before attempting this guide.
```
Comment (0Grit) [inline docs/hardware/5-virtual-device/2-quickstart/1-overview.md:8]: See my quickstart/getting started comment above

File: docs/landing_page.md [hunk 4]
```diff
@@ -23,14 +23,14 @@ We chose the [the Zephyr Project™](https://www.zephyrproject.org), as the basi
 
 ## Get started today
 
-To get started, check out the [Quickstart Guide](/services/getting-started/). That will get you set up with a Golioth account and help you to provision your first devices. Then you can choose which platform to try out using our [supported hardware](/hardware).
+To get started, check out the [Quickstart Guide](/getting-started/). That will get you set up with a Golioth account and help you to provision your first devices. Then you can choose which platform to try out using our [supported hardware](/hardware).
```
Comment (0Grit) [inline docs/landing_page.md:26]: quickstart/getting started

**Review by 0Grit — COMMENTED** [review]
Would numbering the Getting Guide sections ease use as a walkthrough?

**Review by beriberikix — APPROVED** [review]

**Comment by beriberikix** [comment]
> Would numbering the Getting Guide sections ease use as a walkthrough?

I like that idea! But let's get this out first and then we can try a different format. If we go that route we should make sure all other step by step guides are consistent and use numbers.

**Pull request merged**